packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -438,4 +551,47 @@ export class CXLMarketingNavElement extends LitElement {
       el.close();
     }
   }
+
+  /**
+   * Initialize Headroom.
+   *
+   * @since 2022.11.10
+   *
+   * @param contextMenuItems
+   * @param Headroom
+   */
+  async initHeadroom(Headroom) {
+    /**
+     * Fix race condition where the css properties are not available yet.
+     *
+     * @since 2022.11.11
+     * @see https://cxlworld.slack.com/archives/C01JABH8AHX/p1667298030545399?thread_ts=1667211996.651629&cid=C01JABH8AHX
+     * @see https://stackoverflow.com/a/63062857
+     * @see https://lit.dev/docs/components/lifecycle/#updatecomplete
+     */
+    await this.updateComplete;
+
+    const bodyElement = document.querySelector('body');
+
+    /**
+     * Headroom.
+     *
+     * @see https://github.com/WickyNilliams/headroom.js
+     */
+    const headroom = new Headroom(this, {
+      tolerance: {
+        up: 30,
+        down: 30,
+      },
+      offset: this.hasAttribute('wide') ? Math.max(this.offsetHeight, this.clientHeight) / 2 : 0,
+      onNotTop() {
+        bodyElement.classList.add('cxl-marketing-nav-sticky');
+      },
+      onTop() {
+        bodyElement.classList.remove('cxl-marketing-nav-sticky');
+      },
+    });
+
+    headroom.init();
+  }
```

Comment:
See:

https://github.com/conversionxl/aybolit/commit/85dd44e75612a26e00904a05479a6b69690ece8d

Basically it dynamically sets the offset, instead of hardcoding it.

Desktop needs to make primary menu / black part of menu, sticky, so it should only be sticky when upper half is scrolled, so smooth sticky action happens.

Where on mobile, it doesn't need any offset, so `0`.

`cxl-marketing-nav-sticky`

Is to aid in `Add some top padding to the page content (body element) to prevent sudden quick movement.`